.eslintrc.js

```diff
@@ -23,14 +23,15 @@ module.exports = {
     semi: ["error", "always"],
     "@typescript-eslint/camelcase": "off",
     "comma-dangle": ["error", "always-multiline"],
-    indent: ["error", 2],
+    indent: ["off"],
```

Comment:
This `indent` rule can be omitted since the `"prettier/prettier"` rule on the next line does its own indentation checks.

Reply:
Omitting the indent rule is not enough.  I ran into scenarios where Prettier and eslint started conflicting over how the page should appear.

A good example was where I added the switch() statement for the SuiteFactory.  Prettier would make its changes and eslint would flag it as an error.

Reply:
Attached two files showing eslint and prettier conflicting over indents.
Eslint complaining
<img width="482" alt="prettier" src="https://user-images.githubusercontent.com/89038375/188935010-1f2bd202-68d8-45a7-a17e-c1eb34461be6.png">

Prettier complaining
<img width="474" alt="eslint" src="https://user-images.githubusercontent.com/89038375/188935029-ff39fa50-9dc1-4262-9874-619463c0ce5f.png">

I recommend to give prettier control over behavior.